bedrock/firefox/firefox_details.py

```diff
@@ -296,11 +296,11 @@ class FirefoxAndroid(_ProductDetails):
     }
 
     store_url = settings.GOOGLE_PLAY_FIREFOX_LINK
-    aurora_url_base = ('https://ftp.mozilla.org/pub/mozilla.org/mobile/nightly/'
+    aurora_url_base = ('https://archive.mozilla.org/pub/mobile/nightly/'
                        'latest-mozilla-aurora-android')
```

Comment:
I think the URL is built property, but the current version is 46. You probably have to run `bin/sync_all` to update local product-details :smiley: